packages/story-editor/src/components/panels/design/animation/animation.js

```diff
@@ -251,16 +257,21 @@ function AnimationPanel({
             direction={getEffectDirection(updatedAnimations[0])}
             selectedEffectType={updatedAnimations[0]?.type}
             selectButtonStylesOverride={highlight?.focus && styles.OUTLINE}
+            disabled={isFirstPage}
           />
         </StyledRow>
         {updatedAnimations[0] && (
           <EffectPanel
             animation={updatedAnimations[0]}
             onChange={handlePanelChange}
             disabledTypeOptionsMap={disabledTypeOptionsMap}
+            disabled={isFirstPage}
           />
         )}
       </GroupWrapper>
+      {isFirstPage && (
+        <Warning message={DESIGN_COPY.firstPageAnimation.animationPanel} />
```

Comment:
Nit: I'd have probably just inlined the `__()` call here instead of adding it to `DESIGN_COPY`.

Now this adds a dependency between the checklist and the animation panel code, which are otherwise not related.

Reply:
@swissspidy - I did consider that, but Im also working on #11320 which works similar to how the text contrast warning, that also shows a checklist item in the Accessibility section.

Reply:
Hmm okay, I see, the text contrast check indeed does this too. Not ideal but also not the end of the world.